google-fcm/src/main/scala/akka/stream/alpakka/google/firebase/fcm/impl/FcmFlows.scala

```diff
@@ -17,42 +20,38 @@ import scala.concurrent.Future
 @InternalApi
 private[fcm] object FcmFlows {
 
-  private[fcm] def fcmWithData[T](conf: FcmSettings,
-                                  sender: FcmSender): Flow[(FcmNotification, T), (FcmResponse, T), NotUsed] =
+  private[fcm] def fcmWithData[T](sender: FcmSender, maxConcurrentConnections: Int, isTest: Boolean)(
+      implicit settings: GoogleSettings
```

Comment:
Instead of requiring implicit `GoogleSettings` you should use `GoogleAttributes.resolveSettings`